custom-vu/src/main/kotlin/jces1209/vu/page/dashboard/cloud/CloudDashboardPage.kt

```diff
@@ -0,0 +1,25 @@
+package jces1209.vu.page.dashboard.cloud
+
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import jces1209.vu.page.dashboard.DashboardPage
+import org.openqa.selenium.By
+
+class CloudDashboardPage(
+
+    private val jira: WebJira
+) : DashboardPage(
+    driver = jira.driver,
+    uri = jira.base.resolve("/secure/ConfigurePortalPages!default.jspa")) {
+    override fun createDashboard() {
+        driver.findElement(
+            By.xpath("(//button[@data-test-id='directory.dashboards-v2.create-button'])")).click()
+        driver.findElement(
+            By.xpath("(//input[@id='shareable-entity-dialog.input-name'])"))
+            .sendKeys("TestDasboard" + System.currentTimeMillis())
+        driver.findElement(
+            By.xpath("(//button[@type='submit'])")).click()
```

Comment:
```suggestion
            By.cssSelector("button[type='submit']")).click()
```

Reply:
all test uses xpath, and for me it is more attractive. i'd leave as is

Reply:
It's mostly a matter of taste. Which one looks more readable can be more subjective. We can just accept both, because there's no known impact on the results.